src/v/features/feature_table.h

```diff
@@ -151,6 +152,7 @@ constexpr cluster::cluster_version to_cluster_version(release_version rv) {
     case release_version::v24_1_1:
     case release_version::v24_2_1:
     case release_version::v24_3_1:
+    case release_version::v25_1_1:
         return cluster::cluster_version{static_cast<int64_t>(rv)};
     }
     vassert(false, "Invalid release_version");
```

Comment:
probably out of scope here, but can't this function be simplified to just compare with MIN and MAX?